documentation-generator/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>tech.picnic.error-prone-support</groupId>
+        <artifactId>error-prone-support</artifactId>
+        <version>0.6.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>documentation-generator</artifactId>
```

Comment:
The suggestion is pretty good. The current one is IMO more inline with "refaster-compiler" and "refaster-runner". The former is most comparable to what we are doing in this module I would say.

The "-support" suffix to me indicates that others can easily use this for their own use cases / websites. (This may actually hold for _some_ classes of this module 🤔, which is a good thing though and would make `-support` the better option.) OTOH how we use it and set it up is quite specific to our use case, which would make me lean towards the current name. 

Not a very clear line of reasoning TBH, in essence both are fine by me. Was mostly not responding yet maybe others would've some input on this :).

Reply:
Discussed with @Stephan202 and settled on a name. It's the alternative you brought up @nathankooij ! It captures the goal nicely and is similar to some other modules.